src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/onetoplevelclass/InputOneTopLevelClassDeclarationOrder.java

```diff
@@ -16,3 +16,8 @@
         protected final int maxTotalConcurrentTasks = 0;
     }
 }
+
+@interface InputDeclarationOrderAnnotation
+{
```

Comment:
move to previous line

Reply:
I assume this is about the braces placement?

This is extended from an existing input file whose style is already like that, should I change the other braces as well? It'll be a bit inconsistent otherwise.

Reply:
Ok, I didn't see definitions above. Let it be consistent.